docs/developer_guide/env_setup.md

```diff
@@ -75,24 +75,12 @@ Dashboard Web UI. The dashboard may be deployed using the following steps:
 ### Access local Kubernetes cluster
 
 The dashboard as deployed in the previous step requires a token or config to
-sign in. Here we use the token to sign in. The token is generated
-automatically and can be retrieved from the system.
+sign in. Here we use the token to sign in.
```

Comment:
` requires a token or config to sign in` -> `config` is no longer an option in the latest kubernetes dashboard.

Reply:
delete config done